feature-libs/product-configurator/rulebased/components/attribute/types/checkbox/configurator-attribute-checkbox.module.ts

```diff
@@ -4,6 +4,7 @@ import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { I18nModule } from '@spartacus/core';
 import { KeyboardFocusModule } from '@spartacus/storefront';
 import { ConfiguratorAttributeCheckBoxComponent } from './configurator-attribute-checkbox.component';
+import { ConfiguratorPriceModule } from '../../../price/configurator-price.module';
```

Comment:
Why is ConfiguratorPriceModule imported? I can't see any usage.

Reply:
It is imported to be used within the HTML component

Reply:
Otherwise the price component is unknown

Reply:
In configurator-attribute-checkbox.component.html I don't see any usage.

Reply:
I see that the price component was accidentally removed from the HTML

Reply:
It should be added, otherwise the price won't be displayed